README.md

```diff
@@ -40,7 +40,7 @@ Our builds require various development dependencies (e.g. Java, Bazel, Python),
 
 On Linux and Mac `dev-env` can be installed with:
 
-1. Install Nix by running: `bash <(curl https://nixos.org/nix/install)`
+1. Install Nix by running: `bash <(curl -sfL https://nixos.org/nix/install)`
```

Comment:
How do you feel about adding `-S` to this so it shows an error on failure?

Reply:
changed.